src/Entity/Token.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Pucene\Bundle\DoctrineBundle\Entity;
+
+/**
+ * Token entity.
+ */
+class Token
```

Comment:
ok thats clear (: but i think we will have a size problem before we will have any performance problem!